src/main.cpp

```diff
@@ -3670,6 +3670,7 @@ bool static FlushStateToDisk(
     LOCK2(cs_main, cs_LastBlockFile);
     static int64_t nLastWrite = 0;
     static int64_t nLastFlush = 0;
+    static int64_t nCallCount = 0;
```

Comment:
```suggestion
    static int nCallCount = 0;
```
This doesn't matter for the experiment.